server/src/pages/shot/text-tool.js

```diff
@@ -102,11 +105,21 @@ exports.TextTool = class TextTool extends React.Component {
       cursor: "move"
     };
 
+    // Styles used by hidden div that is used to compute width of input
+    // text for respective font-size
+    const hiddenDivStyles = {
+      position: "absolute",
+      zIndex: "-999",
+      opacity: "0",
+      top: "-999px"
```

Comment:
We should add `visibility: hidden` here.

Reply:
with visibility:hidden, we can safely remove opacity and top here